extensions/wikia/HTTPSSupport/tests/HTTPSSupportHooksTest.php

```diff
@@ -86,4 +90,98 @@ public function onBeforeResourceLoaderCSSMinifierDataProvider() {
 			]
 		];
 	}
+	/**
+	 * @param string $url
+	 * @param string $environment
+	 * @param string $expectedResult
+	 *
+	 * @dataProvider onLinkerMakeExternalLinkDataProvider
+	 */
+	public function testOnLinkerMakeExternalLink( $url, $environment, $expectedResult ) {
+
+		$exampleString = '';
+		$exampleBool = false;
+		$exampleArray = [];
+
+		$this->mockEnvironment( $environment );
+		HTTPSSupportHooks::onLinkerMakeExternalLink($url, $exampleString, $exampleBool, $exampleArray);
```

Comment:
you're missing spaces in the parentheses. I've fixed that i few places, but can you please configure your IDE so the code follows the guidelines? https://github.com/Wikia/guidelines/tree/master/PHP